### PR TITLE
9.4.0 Release

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Waiting on:
- [x] https://github.com/cypress-io/cypress/pull/19977

### CODE REVIEW

File: content/_changelogs/9.4.0.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## 9.4.0
+
+_Released 1/31/2022_
+
+**Features**
+
+- Enhancements were made to `.selectFile()` feedback receiving after it's
+  initial release in [9.3.0](/guides/references/changelog#9-3-0).
```
Comment (jennifer-shehane) [inline content/_changelogs/9.4.0.md:8]: You make the same mistake with "its" and "it's" that I do. We're doomed between the two of us now. 😅 

```suggestion
- Enhancements were made to `.selectFile()` after receiving feedback after its
  initial release in [9.3.0](/guides/references/changelog#9-3-0).
```

File: content/_changelogs/9.4.0.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - The default behavior was updated to automatically infers the mime type of
+    files based on their extension to correctly encode file uploads. Addressed
+    in [#19751](https://github.com/cypress-io/cypress/issues/19751).
```
Comment (jennifer-shehane) [inline content/_changelogs/9.4.0.md:11]:
```suggestion
  - The default behavior was updated to automatically infer the mime type of
    files based on their extension to correctly encode file uploads. Addressed
    in [#19751](https://github.com/cypress-io/cypress/issues/19751).
```

File: content/_changelogs/9.4.0.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - Added the `mimeType` property to allow users to explicitly set the mime type
+    on files using the `mimeType` property. Addressed in
+    [#19751](https://github.com/cypress-io/cypress/issues/19751).
```
Comment (jennifer-shehane) [inline content/_changelogs/9.4.0.md:14]:
```suggestion
  - Added the `mimeType` property so you can explicitly set the mime type
    on files using the `mimeType` property. Addressed in
    [#19751](https://github.com/cypress-io/cypress/issues/19751).
```

File: content/_changelogs/9.4.0.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  - Updated file contents types to allow either a
+    [`TypedArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray)
+    instance or a `Cypress.Buffer` instance, where previously file contents were
+    expected to only be an instance of `Cypress.Buffer`. Addressed in
+    [#19751](https://github.com/cypress-io/cypress/issues/19751).
+  - Updated `.selectFile()` to retains the fileName of files read from disk to
+    work with aliases. Addressed in
+    [#19803](https://github.com/cypress-io/cypress/issues/19803).
```
Comment (jennifer-shehane) [inline content/_changelogs/9.4.0.md:22]:
```suggestion
  - Updated `.selectFile()` to retain the fileName of files read from disk to
    work with aliases. Addressed in
    [#19803](https://github.com/cypress-io/cypress/issues/19803).
```

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

File: content/_changelogs/9.4.0.md (hunk 1 above)
Comment (emilyrohrbough) [reply content/_changelogs/9.4.0.md:8]: Oh no! 😢

**Review by jennifer-shehane — APPROVED** [review]
Looks good. Waiting on Node 17 line item as well.

**Comment by emilyrohrbough** [comment]
@jennifer-shehane The Node 17 changes have merged.

**Review by flotwig — APPROVED** [review]

**Review by flotwig — APPROVED** [review]

**Pull request merged**